android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -442,10 +442,33 @@ class ScreenStackFragment :
         }
     }
 
+    private fun isComposeView(view: View): Boolean {
+        val name = view.javaClass.name
+        return name.startsWith("androidx.compose.ui.platform.")
```

Comment:
Found out it doesn't work in release mode - R8 strips class names.

Normally I'd just check if view is instance of a compose class, however here adding compose as a dependency doesn't seem an option. 

Looking forward to some advice on how to tackle this